paddle/fluid/operators/detection_map_op.h

```diff
@@ -76,8 +76,8 @@ class DetectionMAPOpKernel : public framework::OpKernel<T> {
     auto ap_type = GetAPType(ctx.Attr<std::string>("ap_type"));
```

Comment:
Remove a lot of memcpy in this op. Not related to bug fix, however, they will speed up this Op